scattering/van_hove.py

```diff
@@ -257,37 +250,139 @@ def compute_partial_van_hove(
     # Don't need to store it, but this serves to check that dt is constant
     dt = get_dt(trj)
 
+    trj = trj.atom_slice(trj.top.select(str(selection1) + " or " + str(selection2)))
+
     pairs = trj.top.select_pairs(selection1=selection1, selection2=selection2)
 
-    n_chunks = int(trj.n_frames / chunk_length)
+    if chunk_starts is None:
```

Comment:
I think you need to move this line up before `chunk_starts` is iterated on.  I believe this is why tests are currently failing.